src/main/java/dansplugins/factionsystem/services/PlayerService.java

```diff
@@ -0,0 +1,47 @@
+package dansplugins.factionsystem.services;
+
+import dansplugins.factionsystem.MedievalFactions;
+import org.bukkit.ChatColor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.command.ConsoleCommandSender;
+
+import java.util.List;
+
+public class PlayerService {
+
+    public static String getMessageType(String OldType, String NewType) {
+        if (MedievalFactions.USE_NEW_LANGUAGE_FILE) {
+            return NewType;
+        } else {
+            return OldType;
+        }
+    }
+
+    public static void sendMessageType(CommandSender sender, String OldType, String NewType, Boolean replace) {
```

Comment:
Variables should start with a lower case letter. replace can be a primitive boolean, it doesn't need the wrapper class, which allows null values to be passed in

Reply:
replace here means, if true, it will get the message in new message, and if false then just type the first one of the message (I don't know what to call it but it's like getText(), I just type the one inside the quotes) then the method takes that line in the language file of yml to send it to the player, making typing a bit shorter